app/graph/mutations/media_mutations.rb

```diff
@@ -4,8 +4,7 @@ module MediaMutations
     account_id: 'int',
     project_id: 'int',
```

Comment:
`project_id` is part of  `media_mutations` to assign a media to project.
if we remove `Media` mutation then we will need to pass URL or Quote to `ProjectMedia` mutation to create a media object then associate it to `ProjectMedia`